### PR TITLE
Cu 9evxz1 list items as anchor

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This PR aims to append a little improvement to our way to manage links in the code base

- I deleted `<NeutralLink />` story as it was useless
- I moved `<NeutralLink />` component to the root of display component's folder
- I deleted `<NeutralLink />` tag wherever it was used, I preferred extending the child of `<Link />` by `NeutralLink`
- I moved `<BoxedLink />` to `<LoginsOverview />` as its use was very specific to this page
- I deleted `@src/components/display/fewlines/BoxedLink` folder because the story was useless and the component was moved

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
I only modified imports in test files

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx [hunk 1]
```diff
@@ -15,6 +15,11 @@ type LoginsOverviewProps = {
   sortedIdentities: SortedIdentities;
 };
 
+type BoxedLinkProps = {
+  primary: boolean;
+  status: "validated" | "unvalidated";
+};
```
Comment (Fuzznimp) [inline src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx:21]: As `BoxedLinkProps ` uses types from an already existing type (i.e. `Identity`), you should use `Identity` as a base and extract `primary` and `status` from it, to reflect the connection between the two, and help refacto if `status` casing type changed in the future.

I'll let you find how to do this :).

File: src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx [hunk 2]
```diff
@@ -157,3 +156,26 @@ const IdentitySection = styled.div`
 export const Value = styled.p`
   margin-right: 0.5rem;
 `;
+
+export const BoxedLink = styled(NeutralLink)<
+  Pick<BoxedLinkProps, "primary" | "status">
+>`
```
Comment (Fuzznimp) [inline src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx:162]: Hm, you use `Pick`, but you picked every keys from `BoxedLinkProps `. What did you want to do?

File: src/pages/account/logins/index.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ import type { AccessToken } from "@src/@types/oauth2/OAuth2Tokens";
 import { Container } from "@src/components/display/fewlines/Container";
 import { H1 } from "@src/components/display/fewlines/H1/H1";
 import { H2 } from "@src/components/display/fewlines/H2/H2";
-import Logins from "@src/components/display/fewlines/LoginsOverview/LoginsOverview";
+import LoginsOverview from "@src/components/display/fewlines/LoginsOverview/LoginsOverview";
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:11]: Can you remove the `default` export from `LoginsOverview`?

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx (hunk 2 above)
Comment (Hamza-A) [reply src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx]: `Pick` is no longer useful with this refacto, fixed 75920d6

File: src/pages/account/logins/index.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed 75920d6

File: src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx]: What about this solution ? 48027c1

Comment (Fuzznimp) [reply src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx]: Perfect :)

**Review by Fuzznimp — APPROVED** [review]

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**